Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp

```diff
@@ -697,6 +718,15 @@ void Arm64FPRCache::GetAllocationOrder()
 
 void Arm64FPRCache::FlushByHost(ARM64Reg host_reg)
 {
+  for (HostReg& reg : m_host_registers)
```

Comment:
Perhaps a helper function should be introduced that operates like `find_if` for scanning host registers, given this loop pattern is used in multiple places

Reply:
The specific pattern of looking for a host register for which `host_reg.GetReg()` is equal to a particular value only occurs in two places as far as I can tell. Would this helper function be just for those two cases, or more general? If it's the latter, I don't think a helper function would be much more convenient than calling `std::find_if` directly (you would just skip having to write `m_host_registers.begin(), m_host_registers.end()`).